bouter/__init__.py

```diff
@@ -14,3 +15,19 @@ def load_folder(folder, exp_class=Experiment):
     """
     folder = Path(folder)
     return [exp_class(f) for f in sorted(folder.glob("**/*_metadata.json"))]
+
+
+def get_scale_mm(exp_class: Experiment):
```

Comment:
This should be a property (not method) of a freely-swimming experiment

Reply:
Fixed that, but the get_scale_mm method had a clause for embedded experiments. Just to keep in mind in case it was used somewhere (although I couldn't find any other use) and errors might pop up...